app/helpers/application_helper.rb

```diff
@@ -31,4 +31,8 @@ def firm_map_component(center:, adviserPinUrl:, officePinUrl:)
       yield
     end
   end
+
+  def feature_enabled?(flag_name)
+    ENV[flag_name] == 'true'
+  end
```

Comment:
Should this be added to the development environment too?

Alternatively, we could flip the logic so we only have to explicitly disable it in production rather than having to enable it everywhere else. Wdyt?